templates/remote_syslog.systemd.conf.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=remote_syslog2
+Documentation=https://github.com/papertrail/remote_syslog2
+After=network-online.target
+
+[Service]
+ExecStartPre=/usr/bin/test -e {{ papertrail_conf }}
```

Comment:
There needs to be an `-` after the `ExecStartPre =` here or else the unit will fail to start in the event that the file specified by `papertrail_conf` isn't present.